### PR TITLE
Style

### PR DESCRIPTION
**Pull request by nries1**
Changes this PR
 - Added Styling per Cissy's wireframes

### CODE REVIEW

File: app/components/Home.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react';
 
 class Home extends Component {
   render() {
-    return <div> Welcome to CLIMBAR</div>;
+    return <div>Connect Collaborate &amp; Create Amazing Climbing Routes</div>;
```
Comment (wulalasuperdope) [inline app/components/Home.js:5]: We probably want to put this line in the middle of the page and increase the text size :D

File: app/components/Navigation.js [hunk 2]
```diff
@@ -13,45 +13,47 @@ class Navigation extends Component {
...
-              }}
+              title={
+                <span id="nav-dropdown">
+                  {user.email.replace(/@[aA-zZ | . | 0-9]*/, '')}
+                  <i className="material-icons">arrow_drop_down</i>
+                </span>
+              }
             >
               <NavDropdown.Item eventKey="4.1" href={`/profile`}>
                 Profile
```
Comment (wulalasuperdope) [inline app/components/Navigation.js:34]: Maybe put "Profile" in the center of the box?

File: app/components/Navigation.js [hunk 3]
```diff
@@ -96,28 +88,28 @@ class Navigation extends Component {
     const { user } = this.props;
     return (
       <Navbar
-        bg="dark"
+        id="navbar"
         style={{
           height: '7rem',
-          backgroundColor: 'black',
```
Comment (wulalasuperdope) [inline app/components/Navigation.js:102]: I actually didn't mean to change the background color in Navbar. Could you keep it the same as the default setting?

File: app/components/Navigation.js [hunk 4]
```diff
@@ -96,28 +88,28 @@ class Navigation extends Component {
...
+          backgroundColor: '#070707',
           fontWeight: '100',
           padding: '0 5rem'
         }}
       >
-        <Navbar.Brand>Logo</Navbar.Brand>
+        <Navbar.Brand id="logo">Climbar</Navbar.Brand>
```
Comment (wulalasuperdope) [inline app/components/Navigation.js:99]: Can we make the brand size a little bit bigger?

File: app/components/RatingDisplay.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+const RatingDisplay = ({ avgRating }) => {
```
Comment (wulalasuperdope) [inline app/components/RatingDisplay.js:3]: This is dope!

File: app/css/climbingRoutes.css [hunk 6]
```diff
@@ -1,7 +1,7 @@
 .allRoutes {
```
Comment (wulalasuperdope) [inline app/css/climbingRoutes.css:1]: Screen Shot 2020-03-04 at 3 11 45 PM

The 4th card is a small issue when changing the browser's width. Maybe consider setting the maximum width of each card.

**Review by wulalasuperdope — COMMENTED** [review]
The All Climbing Routes page is great! Just added a few small fix.

File: app/components/Navigation.js (hunk 2 above)
Comment (nries1) [reply app/components/Navigation.js:34]: Done

File: app/components/Home.js (hunk 1 above)
Comment (nries1) [reply app/components/Home.js:5]: Done

File: app/components/Navigation.js (hunk 3 above)
Comment (nries1) [reply app/components/Navigation.js:102]: Done

File: app/components/Navigation.js (hunk 4 above)
Comment (nries1) [reply app/components/Navigation.js:99]: Done

File: app/components/RatingDisplay.js (hunk 5 above)
Comment (nries1) [reply app/components/RatingDisplay.js:3]: Thanks!

File: app/css/climbingRoutes.css (hunk 6 above)
Comment (nries1) [reply app/css/climbingRoutes.css:1]: Done

**Review by raymondng1 — APPROVED** [review]
Styling looks awesome!

**Pull request merged**